dev-packages/application-package/src/application-package.ts

```diff
@@ -120,6 +119,27 @@ export class ApplicationPackage {
         return this._extensionPackages;
     }
 
+    /**
+     * All entry points defined by the installed Theia extensions.
+     *
+     * Map of entry point name to script import path.
+     */
+    get extensionEntryPoints(): ReadonlyMap<string, string> {
```

Comment:
I searched through the PR, but I don't see the "extensionEntryPoints" instance var used anywhere 🤷

Reply:
> it only works when bundling browser Blueprint because everywhere we happen to spawn paths written as path.join(__dirname, 'entry-point-name')

I'm not sure I follow your argument here: in one of our extensions in Che we _do_ have a require of the form: `@eclipse-che/....`. Why would that not work?

Reply:
When bundling using Webpack all bundles will live in the same folder, so everyone is only reachable by doing `path.join(__dirname, 'entry-point-name')`. If paths were something like `path.join(__dirname, '..', 'some-folder', 'entry-point-name')` it would cause issues once bundled because it would walk out of the bundle folder. From this point the second pass done with `pkg` will most likely fail too because of the erroneous Webpack bundling.